openshift/koku-template.yaml

```diff
@@ -190,14 +279,14 @@ objects:
             valueFrom:
               secretKeyRef:
                 key: database-user
-                name: koku
+                name: koku-secret
           - name: POSTGRESQL_PASSWORD
             valueFrom:
               secretKeyRef:
                 key: database-password
-                name: koku
+                name: koku-secret
```

Comment:
Should we be using `koku-secret` here or `${NAME}-secret`? Same question for the items above.

Reply:
i've gone back and forth on that. my suggestion is to go for this as-is and give it some time to bake, then we can circle back later to further abstract it.